pkg/sm/types.go

```diff
@@ -0,0 +1,29 @@
+package sm
+
+//TODO: once SM changes are merged we should import the rest types from SM here and use them to avoid double maintance
+
+// BrokerList broker struct
+type BrokerList struct {
+	Brokers []Broker `json:"service_brokers"`
+}
+
+// Broker broker struct
+type Broker struct {
+	ID          string       `json:"id"`
+	Name        string       `json:"name"`
+	Description string       `json:"description"`
+	BrokerURL   string       `json:"broker_url"`
+	Credentials *Credentials `json:"credentials,omitempty"`
+	SpaceGUID   string       `json:"space_guid,omitempty"`
```

Comment:
This can be Metadata map[string]interface{} and leave the platform-specific implementation handle it